docs/_vulnerabilities/vulnerabilities.json

```diff
@@ -112,8 +112,24 @@
     ],
     "introduced": "v1.10.1",
     "fixed": "v1.10.6",
-    "published": "2020-12-10",
+    "published": "2020-07-22",
```

Comment:
2021?

Reply:
Ah, damnit, now I have to do the whole signing-dance again.. except felix merged it, so I guess we'll update it next time :)